src/main/drivers/flash_m25p16.h

```diff
@@ -22,6 +22,104 @@
 
 #include "flash_impl.h"
 
+// JEDEC IDs and clock speed for supported devices
+
+// Datasheet: https://docs.rs-online.com/5c85/0900766b814ac6f9.pdf
+#define JEDEC_ID_MACRONIX_MX25L3206E   0xC22016
```

Comment:
Not sure why all of these should be part of the public API for flash chips - they look like internals to me and should probably be inside of the unit, not outside.